config/routes.js

```diff
@@ -91,7 +91,8 @@ module.exports.routes = {
   '/api/game/over': 'game/game-over',
   '/api/game/concede': 'game/concede',
   '/api/game/stalemate': 'game/stalemate',
-  '/api/game/reject-stalemate': 'game/stalemate-reject',
+  '/api/game/stalemate-accept': 'game/stalemate',
```

Comment:
The legacy API still only uses one `game/stalemate` action for requesting + accepting stalemates, but the GameState API treats it as two moveTypes. Adding this redundant route for the legacy API to point to the same `game/stalemate` action allows consistent url transformation between API's. Should remove this on cleanup though along with all the other `/api/game/X` endpoints